README.md

```diff
@@ -19,7 +19,7 @@ The following is the list of supported operating systems.
 |--------------------|--------------|--------------|
 | redis_7            |      x       |       x      |
 | mysql_8_4          |      x       |       x      |
-| mongo_7_0          |      x       |              |
+| mongo_7_0          |              |       x      |
```

Comment:
What happend to 22.04 support?

Reply:
break system packages is not support in ubuntu 22

`fatal: [mongo-7-0-ubuntu2204]: FAILED! => {"changed": false, "cmd": ["/usr/bin/python3.10", "-m", "pip.__main__", "install", "-i", "https://pypi.python.org/simple", "--break-system-packages", "pymongo==4.5.0"], "msg": "\n:stderr: \nUsage:   \n  /usr/bin/python3.10 -m pip install [options] <requirement specifier> [package-index-options] ...\n  /usr/bin/python3.10 -m pip install [options] -r <requirements file> [package-index-options] ...\n  /usr/bin/python3.10 -m pip install [options] [-e] <vcs project url> ...\n  /usr/bin/python3.10 -m pip install [options] [-e] <local project path> ...\n  /usr/bin/python3.10 -m pip install [options] <archive url/path> ...\n\nno such option: --break-system-packages\n"}`

Reply:
we could add a check for the ubuntu version and add the flag if its >=24. maybe add it as a variable in `defaults/main.yaml` like `PIP_EXTRA_ARGS`

Reply:
Done